internal/validation/duplicates.go

```
@@ -0,0 +1,237 @@
+		removeDuplicateBlocks(conn, chainId, duplicateBlockNumbers)
+	}
+
+	duplicateTransactions, err := findDuplicateTransactionsInRange(conn, chainId, startBlock, endBlock)
+	if err != nil {
+		return err
+	}
+	if len(duplicateTransactions) == 0 {
+		log.Debug().Msg("No duplicate transactions found in range")
+	} else {
+		log.Debug().Msgf("Found %d duplicate transactions in range %v-%v: %v", len(duplicateTransactions), startBlock, endBlock, duplicateTransactions)
+		removeDuplicateTransactions(conn, chainId, duplicateTransactions)
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)